### ISSUE TITLE
ask for help for KITTI dataset

### ISSUE THREAD

**Issue by 25043221** [RicardooYoung/VersatileTeacher#5]
Excuse me, Can you share your Kitti dataset? As far as I know, the 2D object detection training set on Kitti's official website has over 7000 images, and you mentioned in your article that 4717 images were used. How was it specifically divided?

**Comment by RicardooYoung** [comment]
There is no annotation for the official testing set, so we randomly divided the training set into roughly a 6:1:3 ratio for training, evaluation, and testing, respectively.

I hope this reply helps resolve your issue.

**Comment by 25043221** [comment]
Thank you for your reply!
I understand now.

**Comment by 25043221** [comment]
Excuse me，Can you share the link to your kitti dataset, my experiments result is lower than the metrics in the paper and I don't know what's causing it.

**Comment by RicardooYoung** [comment]
These two .txt files contain images in training set and validation set, respectively. You may split the dataset in the same way as in the paper.

[train.txt](https://github.com/user-attachments/files/18822636/train.txt)

[val.txt](https://github.com/user-attachments/files/18822649/val.txt)
